Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1045,33 +1083,40 @@ KeyCode MapKey (WindowsConsole.ConsoleKeyInfoEx keyInfoEx)
 				return ConsoleKeyMapping.MapToKeyCodeModifiers (keyInfo.Modifiers, (KeyCode)((uint)KeyCode.D0 + delta));
 			}
 			if ((keyInfo.Modifiers & (ConsoleModifiers.Alt | ConsoleModifiers.Control)) != 0) {
```

Comment:
This statement isn't doing what is supposed to do that is only execute if modifiers are `Ctrl+Alt`.
This isn't correct because it will return true if we press Ctrl+Shift+D1.

![image](https://github.com/BDisp/Terminal.Gui/assets/13117724/89122f43-9125-4f3f-af66-691c9ca9ee20)

Reply:
Sorry I was wrong here. It have to execute anything that has the Ctrl key.